src/commands/feedback.js

```diff
@@ -5,7 +5,7 @@ const { Command } = require('@oclif/command');
 class Feedback extends Command {
   async run() {
     const link =
-      chalk.bold('>>>>>  ') + chalk.underline.bold('https://twil.io/twilio-cli-feedback') + chalk.bold('  <<<<<');
+      chalk.bold('>>>>>  ') + chalk.underline.bold('https://airtable.com/shrcFDU1gmKWOqZXe') + chalk.bold('  <<<<<');
```

Comment:
Can we make this link clickable with the hyperlink utility? Makes it easier for the user to navigate

Reply:
Added this